pkg/oc/cli/admin/admin.go

```diff
@@ -127,9 +126,7 @@ func NewCommandAdmin(name, fullName string, f kcmdutil.Factory, streams genericc
 	// Deprecated commands that are bundled with the binary but not displayed to end users directly
 	deprecatedCommands := []*cobra.Command{
 		// these will be removed soon
-		admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, streams),
 		node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, streams),
-		router.NewCmdRouter(f, fullName, "router", streams),
 		ipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", streams),
```

Comment:
@smarterclayton do we want to get rid of these and below ca-related commands too?